### PR TITLE
Set PHP version to 5.3 in pantheon.upstream.yml.

### PR DESCRIPTION
**Pull request by greg-1-anderson**
Presently, all new Pantheon sites default to PHP 5.5; however, sites that were created prior to oh-I-can't-find-when-but-probably-2015-something will still converge to PHP 5.3 based on some state that ygg stores in Cassandra. There used to be a way to change this state in the debug pane of the admin dashboard, but that affordance has long since been removed. As a result, this state is set to 5.5 for most sites, or to 5.3 for a few old sites, and is overridden almost everywhere by `pantheon.upstream.yml` (provided by drops-8, drops-7 and wordpress/wordpress-network) or `pantheon.yml` (if added by the customer).

This PR adds a `pantheon.upstream.yml` to drops-6, which would make this PHP state variable irrelevant to at least those Drupal 6 sites that update their site through the dashboard after this PR is merged. This would get a lot closer to allowing us to eliminate that PHP state variable, and rely only on pantheon*.yml. There are probably still a bunch of old custom upstreams that would have to be chased down and fixed (notify customer to fix) to get there, though.

### CODE REVIEW

**Comment by greg-1-anderson** [comment]
Drupal 6 reached EOL on February 24th 2016. I am not sure when it was removed as an option from the new site framework selection panel on the dashboard, but there are potentially a lot of d6 on the platform that might be running php 5.5 without a pantheon.yml file.

I'm therefore thinking that perhaps this PR should be modified to set PHP 5.5 by default. All of the folks who were broken before will need to add a pantheon.yml file to stay on PHP 5.3 iff they merge this PR. Of course merging this PR will happen on a multidev or dev environment, so users will have  chance to test before pushing to test & live.

In short, I'd rather inadvertently force a few people to go UP in version than to inadvertently downgrade a few people.

**Comment by greg-1-anderson** [comment]
Closed in favor of #22.
